src/commands/run.ts

```diff
@@ -1,23 +1,31 @@
 import { RunOptions } from "../types";
 import { getLogger, DBService } from "../utils";
 import { ChainContext } from "../domain";
+import { ApiService } from "../utils/api";
 
 /**
  * Run the watch-tower 👀🐮
  * @param options Specified by the CLI / environment for running the watch-tower
  */
 export async function run(options: RunOptions) {
   const log = getLogger("commands:run");
-  const { rpc, deploymentBlock, oneShot } = options;
+  const { rpc, deploymentBlock, oneShot, enableApi, apiPort } = options;
+
+  const api = new ApiService(apiPort);
```

Comment:
can we conditionally create this api by option (default could be to create)

Reply:
Done. Instead now it's specifically required to do `--disable-api`. The objects etc are only created in the event that it's not explicitly disabled.